src/QueryRenderer.ts

```diff
@@ -181,7 +187,6 @@ class QueryRenderChild extends MarkdownRenderChild {
 
         return { taskList, tasksCount };
     }
-
```

Comment:
By accident?

Reply:
Oh - the removing of the end-of-line. Yes, by accident. Will fix.

Reply:
Fixed